src/PHPFUI/MySQLSlowQuery/Parser.php

```diff
@@ -138,54 +136,110 @@ private function parse() : void
 			throw new Exception\EmptyLog(self::class . ': ' . $this->fileName . ' appears to not exist or is empty');
 			}
 
-		$currentSession = [];
+		$newSessionHeader = [];
+		$parseMode = '';
+		$previousTimeLine = '';
+		$processingSessionHeader = true;
 
+		// Get line from stack (if pushed in below loop) or file. Any comment line
+		// not starting with "# Time: " gets discarded.
 		while (\strlen($line = $this->getNextLine()))
 			{
-			if (0 === \stripos($line, self::PORT))	// in middle of session, end it
```

Comment:
Obsolete comment: "in middle of session, end it" is actually line 224-235. (That is: has always been there, since the initial repository commit.)